roles/st2/tasks/main.yml

```diff
@@ -1,17 +1,25 @@
 ---
-- name: Install latest st2 package
+- name: Install libffi-devel on EL6
   become: yes
-  apt:
+  package:
+    name: ftp://fr2.rpmfind.net/linux/centos/6/os/x86_64/Packages/libffi-devel-3.0.5-3.2.el6.x86_64.rpm
+    state: latest
```

Comment:
Fixed in https://github.com/StackStorm/ansible-st2/pull/66/commits/7fa3a70832051ce26dc981304beb20ca9db6b069